dev/docker/README.md

```diff
@@ -0,0 +1,86 @@
+Apache REEF (incubator) Docker Tests
+====================================
+
+[Apache REEF (incubator)](http://reef.incubator.apache.org/) is 
+an approach to simplify and unify the lower layers of big data systems
+on modern resource managers. This project maintains docker images
+to simulate the target underlying systems as similar as possible.
+
+Docker-based Test Cluster
+-------------------------
+
+| OS           | Java            | Hadoop Version            | Description   | alias     |
```

Comment:
I see. I was just curious because we don't seem to test HDInsight in `reef-tests`(although I'm not sure whether that's necessary). But anyways that is out of the scope of this issue.

Yes please see whether you can fit the new field into the current table. If not possible, I'm fine with splitting it.